src/js/components/registerComp.js

```diff
@@ -0,0 +1,58 @@
+import { newContainer, newText, newMultimedia } from "../helpers/helpers";
+
+const createInputs = (formInputs) => {
+  const arrayInputs = formInputs.map((input) => {
+
+    const formFieldLabel = newText('label', input.label, ['form__field-label']);
+
+    const formFieldInput = document.createElement('input');
+    formFieldInput.classList.add('form__field-input');
+    formFieldInput.type = input.type;
+    formFieldInput.placeholder = input.placeholder;
+
+    const formField = newContainer('div', [formFieldLabel, formFieldInput], ['form__field']);
+
+    return formField;
+  });
+
+  return arrayInputs;
+
+};
+
+const registerComp = (data) => {
+
+  const formHeadline = newText('h6', data.headline, ['form__headline']);
+  const formSubheadline = newText('p', '', ['form__subheadline']);
+  //This is a "hack" to create a tag <br> within the subheadlines data
+  formSubheadline.innerHTML = data.subheadline;
```

Comment:
If you need that a text breaks into two lines, please use CSS to do so. You can set, for instance, in mobile a fixed `width` or `max-width` and remove it in desktop viewport.

Reply:
I tried that way but the flow of the text doesn't break like the one presented in the designs if I set a max-width for it.